base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
Julia runs on non posix systems too, where uname has no meaning. We aren't calling NT syscalls, we're calling the win32 os api. If we built in cygwin the kernel would still be NT but we'd want to use unix behaviors. Same in WSL. I don't think we expose any linux syscalls at the Julia level either, we either call into posix libc or call public libuv functions which generally go by the libc name rather than the kernel specific details.

`uname` is most often executed from shell scripts, so people who don't write code in that kind of environment aren't going to be familiar with that name.